src/test/java/persistence/study/proxy/HelloTargetLazyLoaderTest.java

```diff
@@ -0,0 +1,40 @@
+package persistence.study.proxy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import net.sf.cglib.proxy.Enhancer;
+import net.sf.cglib.proxy.LazyLoader;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+@DisplayName("LazyLoader CallBack 테스트")
+public class HelloTargetLazyLoaderTest {
+
+    @Test
+    @DisplayName("Enhancer를 통해 컬렉션 필드를 LazyLoad하는 Proxy를 생성할 수 있다.")
+    public void canLazyLoadToUppercaseProxy() {
+        final Enhancer enhancer = new Enhancer();
+
+        enhancer.setSuperclass(List.class);
+        enhancer.setCallback(new UppercaseLazyLoader());
+
+        final List<Address> lazyLoadedAddress = (List<Address>) enhancer.create();
+
+        final HelloTarget helloTarget = new HelloTarget(lazyLoadedAddress);
+
+        assertThat(helloTarget.getAddress().size()).isEqualTo(2);
```

Comment:
```suggestion
        assertThat(helloTarget.isCalled()).isFalse();
        assertThat(helloTarget.getAddress().size()).isEqualTo(2);
        assertThat(helloTarget.isCalled()).isTrue();
```
상태 추가하고 이렇게 하면 조금 더 유의미한 테스트가 될 것 같다고 생각합니다